### PR TITLE
[chore: 162727504]Setup react router

### PR DESCRIPTION
**Pull request by XROLE**
#### What does this PR do?
- Setup routing functionality

#### Description of task to be completed?
- [x] install react-router-dom
- [x] add about page component
- [x] add devser configuration to webpack
- [x] configure App.js file for routing

#### How should this be tested
- run `npm install` and `npm run start:dev`

#### What are the relevant pivotal tracker stories?
[#162727504](https://www.pivotaltracker.com/story/show/162727504)

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,9 +1,16 @@
 import React from 'react';
+import {BrowserRouter as Router, Route, Switch} from 'react-router-dom';
```
Comment (andela-mabdussalam) [inline src/components/App.js]: This is not properly formatted.

Comment (XROLE) [reply src/components/App.js]: Ok. Fixing that now

File: package.json [hunk 2]
```diff
@@ -22,9 +22,9 @@
     },
     "homepage": "https://github.com/andela/pippin-ah-frontend#readme",
     "dependencies": {
+        "express": "^4.16.4",
```
Comment (andela-mabdussalam) [inline package.json]: why did you move this up?

Comment (XROLE) [reply package.json]: No concrete reason, just that it occured twice so I deleted one and moved the other

**Comment by andela-mabdussalam** [comment]
@XROLE Remove the component you added.  We no longer need it. Also add `Switch` back as a wrapper to the routes. It renders the routes exclusively.

**Review by andela-mabdussalam — APPROVED** [review]
